bin/rialto/node/src/service.rs

```diff
@@ -713,6 +732,22 @@ where
 	let keystore_opt =
 		if role.is_authority() { Some(keystore_container.sync_keystore()) } else { None };
 
+	let beefy_params = beefy_gadget::BeefyParams {
+		client: client.clone(),
+		backend: backend.clone(),
+		key_store: keystore_opt.clone(),
+		network: network.clone(),
+		signed_commitment_sender,
+		min_block_delta: 4,
```

Comment:
```suggestion
		min_block_delta: 2,
```

Let's make it faster.